lib/jekyll-redirect-from/redirector.rb

```diff
@@ -82,5 +82,9 @@ def config_github_url(site)
     def config_baseurl(site)
       site.config.fetch('baseurl', nil)
     end
+
+    def config_url(site)
+      site.config.fetch('url', nil)
```

Comment:
Actually to simplify, why not make this `URI.join site.config.fetch('url', nil), config_baseurl(site)` and then we can always assume anything that ends in `_url` will be the full URL with basepath (and above just becomes `config_github_url || config_url`